conf/modules/short_variant_calling.config

```diff
@@ -25,8 +25,8 @@ process {
     }
 
     withName: '.*:SHORT_VARIANT_CALLING:DEEPVARIANT' {
-
-        ext.prefix = { intervals ? "${meta.id}" + "_" + "${intervals.getSimpleName()}" + "_deepvariant" : "${meta.id}" + "_deepvariant"}
+        // Needed to not cause input name collision in bcftools concat
+        ext.prefix = { intervals ? "${meta.id}" + "_" + "${intervals}" + "_deepvariant" : "${meta.id}" + "_deepvariant"}
```

Comment:
I think you could simplify to this of you prefer:
`intervals ? "${meta.id}_${intervals}_deepvariant" : "${meta.id}_deepvariant"}`

Reply:
Yes, much simpler! 🤦

Reply:
I'll also add a test without intervals so this is tested.